src/main/resources/application.properties

```diff
@@ -4,3 +4,4 @@ uk.gov.companieshouse.item-handler.health: /healthcheck
 uk.gov.companieshouse.item-handler.error-consumer: ${IS_ERROR_QUEUE_CONSUMER}
 
 kafka.broker.addresses: ${KAFKA_BROKER_ADDR}
+spring.kafka.consumer.bootstrap-servers: ${KAFKA_BROKER_ADDR}
```

Comment:
I suppose this might result from the hybrid Spring + CHS Kafka library approach taken? Has it been tested on Vagrant?
You could consider rolling this up into a single configuration property, as these properties have the same value?

Reply:
Initially, I'd kept them separate for clarity, but it does make sense to combine them.

Yes, this works on vagrant.